css/properties/line-break.json

```diff
@@ -79,24 +79,22 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "73"
               },
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤72"
-              },
-              "firefox_android": {
-                "version_added": "79"
+                "version_added": "69"
```

Comment:
✅ Confirmed with BrowserStack Live.

Firefox 68 doesn't support it, but 69 does.